db/migrate/20180423232334_create_feedbook_employee_has_sessions.rb

```diff
@@ -0,0 +1,15 @@
+class CreateFeedbookEmployeeHasSessions < ActiveRecord::Migration[5.1]
+  def change
+    create_table :feedbook_employee_has_sessions do |t|
```

Comment:
Wouldn't it be better to name the join table `:feedbook_employee_sessions` ? Or do you have a reason to add the `has`?